Upgraded NextJS version

pepermao opened the pull request.

pepermao committed:
Upgraded nextjs version from 12.1.0 to 14.2.5
Remove useless nextjs configurations
Start sending query object as a json string, and at the pages parse it to json again. This was necessary after the nextjs version upgraded

thesocialdev commented on server/home/home.controller.ts:
```diff
@@ -47,6 +48,7 @@ export class HomeController {
             },
             latest: true,
         });
+        console.log("review", reviews);
```
chore: remove console.log

thesocialdev reviewed (approved):
once the console.log is removed, it can be merged

thesocialdev commented on server/badge/badge.controller.ts:
```diff
@@ -151,15 +151,16 @@ export class BadgeController {
+            nameSpace: req.params.namespace,
+        });
+
+        await this.viewService
```
suggestion: we should create an abstraction in the viewService that receive the parameters from the controllers but perform the serialization and rendering in a single function

pepermao committed:
remove console.log and typescript configurations

pepermao committed:
Created an abstraction function that performs the query serialization and render next in one function

pepermao committed:
Try fix pipeline build

thesocialdev committed:
Simplify nextjs build and fix Docker build

- Remove unnecessary packages
- Stop using babel
- Bump specific versions
- Transpile antd packages
- Fix Dockerfile missing files

thesocialdev committed:
Remove the export to fix 400 error as in https://github.com/ory/integrations/issues/29\#issuecomment-2206991583

thesocialdev committed:
Fix how we request sessions/whoami to avoid nextjs rendering

thesocialdev committed:
Remove cache control from sessions/whoami endpoint to avoid cypress problems

thesocialdev committed:
Fix personality and review cypress tests

thesocialdev committed:
Bump cypress command timeout to 100000ms

thesocialdev committed:
Avoid GET /user/:id breakage

thesocialdev committed:
Merge branch 'stage' of github.com:AletheiaFact/aletheia into bump-next-14.1.1-version

thesocialdev committed:
Fix AdminUserMock typescript error

thesocialdev committed:
Bump cypress command timeout to 480000ms

thesocialdev committed:
Bump cypress to a new version and hope for the best
